src/DevSoc.Eventathon.Attendance/AttendanceService.cs

```diff
@@ -0,0 +1,31 @@
+﻿using DevSoc.Eventathon.Attendance.Models;
+using DevSoc.Eventathon.Calendars.Models;
+
+namespace DevSoc.Eventathon.Attendance;
+
+public class AttendanceService : IAttendanceService
+{
+    private readonly IAttendanceRepository _attendanceRepository;
+
+    public AttendanceService(IAttendanceRepository attendanceRepository)
+    {
+        _attendanceRepository = attendanceRepository;
+    }
+
+
+    public void RegisterAttendance(AttendanceDefinition definition)
+    {
+        var attendanceData = new AttendanceData
+        {
+            UserId = definition.UserId,
+            EventId = definition.EventId
+        };
+
+        _attendanceRepository.RegisterAttendance(attendanceData);
```

Comment:
Need to `await` here, so should be an `async` method and return a `Task` here too.